Hardware/sabreLite.md

```diff
@@ -182,4 +182,4 @@ This assumes that the U-Boot version above is installed in SPI flash.
 
 ## Running seL4 test
 
-{% include sel4test.md %}
\ No newline at end of file
+{% include sel4test.md %}
```

Comment:
interesting, what changed here?

Reply:
I suspect my editor removed trailing whitespace, let me check

Reply:
Oh, no my editor wasn't involved, this was via the GitHub interface. So I'm not sure actually.

Reply:
Looking at it in `git diff`, it added a newline at the end of the last line, which I approve of.